tests/backup/backup_restore_basic_test.go

```diff
@@ -1442,3 +1446,159 @@ var _ = Describe("{BackupMultipleNsWithSameLabel}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// MultipleCustomRestoreSameTimeDiffStorageClassMapping issues multiple custom restores at the same time using different storage class mapping
+var _ = Describe("{MultipleCustomRestoreSameTimeDiffStorageClassMapping}", func() {
+	var (
+		contexts          []*scheduler.Context
+		appContexts       []*scheduler.Context
+		bkpNamespaces     []string
+		clusterUid        string
+		clusterStatus     api.ClusterInfo_StatusInfo_Status
+		backupLocationUID string
+		cloudCredName     string
+		cloudCredUID      string
+		bkpLocationName   string
+		backupName        string
+		restoreList       []string
+		sourceScName      *storageApi.StorageClass
+		scNames           []string
+	)
+	namespaceMap := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	labelSelectors := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	storageClassMapping := make(map[string]string)
+	k8sStorage := storage.Instance()
+	params := make(map[string]string)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("IssueMultipleCustomRestoreAtSameTimeWithDiffStorageClassMapping",
```

Comment:
Change the string to `MultipleCustomRestoreSameTimeDiffStorageClassMapping` to keep it consistent with the `Describe` string